tests/phpunit/api/v3/ContributionTest.php

```diff
@@ -2485,12 +2485,6 @@ public function testCompleteTransactionMembershipPriceSet() {
     $logs = $this->callAPISuccess('MembershipLog', 'get', array(
       'membership_id' => $this->_ids['membership'],
     ));
-    //CRM-19600: Ensure that 'Membership Renewal' activity is created after successful membership regsitration
```

Comment:
Can you clarify this for me. I'm guessing you have decided the test was testing behaviour that was incorrect?